external/fv3fit/tests/training/test_main.py

```diff
@@ -424,9 +431,18 @@ def test_cli(
         hyperparameter_dict,
         use_validation_data,
         use_local_download_path=use_local_download_path,
+        unstacked_dims=["z"],
     )
-    mock_load_batches.return_value = [config.mock_dataset for _ in range(6)]
-    cli_main(config.args)
-    fv3fit.load(config.args.output_path)
-    if use_local_download_path:
-        assert len(os.listdir(config.args.local_download_path)) > 0
+    if not use_local_download_path and model_type == "dense":
+        # dense requires caching if unstacked_dims is set as double-stacking
+        # a MultiIndex causes an error, can remove this block when dense training
+        # no longer stacks
```

Comment:
Note this exception only happens if no local download path is set, so we won't see this with argo in practice.

Reply:
Actually, I think by default no local download path is passed in the template, so it's possible.

Reply:
OK, in practice I haven't seen any `run.sh` examples where people haven't set this - should I post in slack about it? This issue should go away within a month or so.

Reply:
Yeah, that sounds fine.